docs/syntax/admonitions.md

```diff
@@ -0,0 +1,148 @@
+# Admonitions
+
+Admonitions (also known as callouts) highlight a particular block of text,
+that exists slightly apart from the narrative of your page, such as a note or a warning.
+
+Admonitions are a special case of {{directive}} extensions.
+It is advised to use admonitions with the [colon_fence](#syntax/colon_fence) extension, which signify that the content of the block is also MyST Markdown.
+
+::::{myst-example}
+
+:::{tip}
+Let's give readers a helpful hint!
+:::
+
+::::
+
+## Admonition types
+
+The following core admonition types are available:
+
+```{myst-admonitions} attention, caution, danger, error, hint, important, note, seealso, tip, warning
+```
+
+These admonitions take no argument, but may be specified with options:
+
+:class: A space-separated list of CSS classes to add to the admonition.
+:name: A reference target for the admonition (see [cross-referencing](#syntax/referencing)).
+
+::::{myst-example}
+
+:::{tip}
+:class: myclass1,myclass2
+:name: a-tip-reference
+Let's give readers a helpful hint!
+:::
+
+[Reference to my tip](#a-tip-reference)
+
+::::
+
+Sphinx also adds a number of additional admonition types, for denoting changes to the documentation, or to the codebase:
+
+::::{myst-example}
+
+:::{versionadded} 1.2.3
+Explanation of the new feature.
+:::
+
+:::{versionchanged} 1.2.3
+Explanation of the change.
+:::
+
+:::{deprecated} 1.2.3
+Explanation of the deprecation.
+:::
+
+::::
+
+## Admonition titles
+
+To provide a custom title for an admonition, use the `admonition` directive.
+If you also want to style the admonition as one of the core admonition types,
+you can use the `admonition` directive with the `class` option.
+
+::::{myst-example}
+
+:::{admonition} My custom title with *Markdown*!
+:class: tip
+
+This is a custom title for a tip admonition.
+:::
+
+::::
+
+## Short form admonitions
+
+Using the [attrs_block](#syntax/attributes/block) extension,
+a single paragraph can be wrapped in an admonition, if the first class matches an admonition name.
+The admonition can also be given a custom title.
+
+:::{myst-example}
+
+{.note}
+This is a note admonition.
+
+{.warning title="This is a *warning*!"}
+This is a warning admonition.
+
+:::
```

Comment:
(didn't hit save, woops.)

Can we put a warning that this might change in the future please. I think we need a MEP to define how the block attributes and directive kinds interact.

My concern is that there are already many ways to create admonitions in various tools, I would rather not add another to the mix. For class-names, I could certainly see using the `warning` class in CSS, and being surprised that this syntax sometimes turns into an admonition.

Reply:
As per #719, I was indeed looking to cherry-pick this implementation out into a separate PR, to discuss 👍

Reply:
yep I've removed this